test/cookbooks/test/recipes/dhcp_host.rb

```diff
@@ -10,3 +10,16 @@
   identifier 'host-identifier option dhcp6.client-id 00:53:00:00:00:01:a4:65:b7:c8'
   address '2001:db8:1:1:0:0:1:10'
 end
+
+# dhcp_host 'Test-Im-A-Bad-IPv4-Host' do
```

Comment:
is there a reason for all the commented out stuff?

Failure tests are better in unit tests imo

Reply:
The failure comes from running the config test so it can't go into spec (at least not easily and stubbing it won't prove anything anyway), I've left them in for convenience but they can be deleted if required.

Reply:
Yeah, I'd just remove them. We're probably never going to uncomment them, and if we do we'll inevitably do it differently, because _reasons_

Reply:
Done